libs/remixd/src/services/slitherClient.ts

```diff
@@ -25,6 +25,28 @@ export class SlitherClient extends PluginClient {
     this.currentSharedFolder = currentSharedFolder
   }
 
+  install (): void {
+    try {
+      console.log('\x1b[32m%s\x1b[0m', `[Slither Installation]: slither requires Python3.6+ (pip3) and solc, the Solidity compiler. They should be installed on your system`)
```

Comment:
I don't know if this is possible but we could publish a docker that'd contains all the dependencies. And retrieve and run that docker from here.

Reply:
mainly because we shouldn't ask the user to install manually pip3 and solc.

Reply:
solc requirement is removed